integreat_cms/cms/models/pages/page.py

```diff
@@ -351,6 +355,15 @@ def restore(self):
                     # The post_save signal will create link objects from the content
                     translation.save(update_timestamp=False)
 
+    @cached_property
+    def mirroring_live_pages(self):
+        """
+        Returns live pages that are mirroring the given page. Archived pages are excluded.
+        """
+        all_mirroring_pages = self.mirroring_pages.all()
+        archived_page_ids = [page.id for page in all_mirroring_pages if page.archived]
+        return all_mirroring_pages.exclude(id__in=archived_page_ids)
```

Comment:
>> This looks mostly good to me. I only wonder whether adding mirroring_live_pages is the best option. Since this pr makes sure that there can't be archived pages which mirror another page, filtering archived pages out of the mirroring pages seems a bit unnecessary to me. We would only have to add a migration that sets mirrored_page to None for all archived pages.
>
> @david-venhoff that's a good idea 😃 If the next reviewes are for it, I'll change the implementation in that way.

I agree as well :innocent: